tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot

```diff
@@ -99,125 +103,92 @@ Verify Alert RHODS-PVC-Usage-At-100 Is Fired When User PVC Is At 100 Percent
 
     [Teardown]    Teardown PVC Alert Test
 
-Verify Alert "RHODS Route Error Burn Rate" Is Fired When Traefik Is Down    # robocop: disable:too-long-test-case
-    [Documentation]    Verifies that alert "RHODS Route Error Burn Rate" is fired when traefik-proxy is not working
+Verify Alerts Are Fired When Traefik Is Down    # robocop: disable:too-long-test-case
+    [Documentation]    Verifies that alerts "RHODS JupyterHub Probe Success Burn Rate" and "RHODS Probe
+    Success Burn Rate" alerts are fired when traefik-proxy is not working
```

Comment:
It should be (3 dots followed by 4 spaces) `...    Success Burn Rate" alerts are fired when traefik-proxy is not working`

Reply:
Updated